src/Export_Command.php

```diff
@@ -382,7 +383,7 @@ private function check_max_file_size( $size ) {
 			return false;
 		}
 
-		$this->max_file_size = $size;
+		$this->max_file_size = $size <= 0 ? WP_EXPORT_NO_SPLIT : $size;
```

Comment:
Let's make `-1` the only supported argument to prevent unexpected, undocumented behavior.

Reply:
what to do with --max-file-size=0 ?
Isn't a zero or negative value explicit enough to keep from spliting dumps?